var/spack/repos/builtin/packages/py-nptyping/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyNptyping(PythonPackage):
```

Comment:
This package seems to be unrelated to py-typish, was this accidentally included in the wrong PR?

Reply:
I've included it here because it depends on py-typish. Will update the PR title to reflect this.